addon/components/imgix-image.js

```diff
@@ -8,16 +8,24 @@ import URI from 'jsuri';
 import { debounce } from '@ember/runloop';
 import { constants, targetWidths } from '../common';
 
+const attributeMap = {
+  src: 'src',
+  srcset: 'srcset',
+  sizes: 'sizes',
+  ...(get(config, 'APP.imgix.attributeNameMap') || {})
```

Comment:
Another day, another extremely useful ES6 pattern that I'm just now learning about.

Reply:
👏👏👏